packages/graphql/src/services/participants.ts

```diff
@@ -486,11 +486,44 @@ export async function flagElement(
     },
   })
 
+  const elementFeedback = await ctx.prisma.elementFeedback.upsert({
+    where: {
+      participantId_elementInstanceId: {
+        participantId: ctx.user.sub,
+        elementInstanceId: args.elementInstanceId,
+      },
+    },
+    create: {
+      feedback: args.content,
+      upvote: false,
+      downvote: false,
+      element: {
+        connect: {
+          id: args.elementId,
+        },
+      },
+      elementInstance: {
+        connect: {
+          id: args.elementInstanceId,
+        },
+      },
+      participant: {
+        connect: {
+          id: ctx.user.sub,
+        },
+      },
+    },
+    update: {
+      feedback: args.content,
+    },
+  })
+
   if (
     !elementInstance?.elementStack.practiceQuiz?.course?.notificationEmail &&
     !elementInstance?.elementStack.microLearning?.course?.notificationEmail
   ) {
-    return null
+    // return early if no notification email has been specified -> only set database entry
+    return 'OK'
```

Comment:
@rschlaefli We should probably return a successful result here, since the feedback has been stored in the database at this point. Before, we simply showed an error toast in the frontend whenever no notification email was specified in the database...

Reply:
yes 👍